### DISCUSSION TITLE
license from chat gpt 😃

### DISCUSSION THREAD

**Discussion by Adamdoesdumbcode** [community/community#142203] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In my latest repo i didnt really find a license that fit what i wanted so i asked chatgpt to make me one does this work.
i know im probably going to sound very stupid right now and my code is trash so i barely need a license anyway but im still curious

**Comment by Chanduporalla** [comment, marked as answer]
It's great that you're thinking about using a license for your project, even if you feel your code isn't perfect—it shows good practice. You don’t sound stupid at all! In fact, adding a license helps clarify how others can use your code, regardless of its current state.

If you had ChatGPT generate a license for you, it would be good to make sure it aligns with your goals. Here are a few things to consider when creating or using a custom license
